toolkit/tools/imagecustomizerapi/verity.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type Verity struct {
```

Comment:
The user shouldn't need to know the UUIDs or labels of the image's partitions to be able to setup dm-verity.

The partition UUIDs of the standard Mariner image will change every time it is built. And the user should be able to update the base image without updating their config. Also, if the config contains partition customizations, then all the UUID could change during partition customization. 

The tool should automatically detect the correct partitions to use. The [findPartitions()](https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/tools/pkg/imagecustomizerlib/partitionutils.go#L24C6-L24C20) function could be useful for this.